client/landing/gutenboarding/onboarding-block/design-selector/index.tsx

```diff
@@ -131,7 +132,7 @@ const DesignSelector: FunctionComponent< Props > = ( { showPageSelector = false
 							onClick={ () => {
 								window.scrollTo( 0, 0 );
 								setSelectedDesign( design );
-								history.push( Step.PageSelection );
+								history.push( makePath( Step.PageSelection, lang ) );
```

Comment:
When redirecting and navigating internally I don't really see the value of adding the `lang` param.